views/mattermost/server_daily_details.view.lkml

```diff
@@ -374,7 +374,7 @@ view: server_daily_details {
     description: "The server's IP Address."
     type: string
     sql: ${TABLE}.ip_address ;;
-    hidden: yes
+    hidden: no
```

Comment:
Would we want to keep these hidden?

Reply:
It doesn't matter if its hidden or not, but I want to unhide them.